src/cloudant/replicator.py

```diff
@@ -163,14 +163,11 @@ def update_state():
             Retrieves the replication state.
             """
             try:
-                repl_doc = self.database[repl_id]
-                repl_doc.fetch()
-                state = repl_doc.get('_replication_state')
+                arepl_doc = self.database[repl_id]
+                arepl_doc.fetch()
+                return arepl_doc, arepl_doc.get('_replication_state')
             except KeyError:
-                repl_doc = None
-                state = None
-
-            return repl_doc, state
+                return None, None
```

Comment:
Any reason why you chose to rewrite this try block?  It doesn't appear to have any py2 v py3 issues.  Also why the renaming of `repl_doc` to `arepl_doc`?